### ISSUE TITLE
Pom goal destroys poms that use shade relocation

### ISSUE THREAD

**Issue by johnoliver** [mojohaus/tidy-maven-plugin#8]
The SectionSorter defined at https://github.com/mojohaus/tidy-maven-plugin/blob/master/src/main/java/org/codehaus/mojo/tidy/task/EnsureOrderAndIndent.java#L70 incorrectly matches "relocation" sections of the pom such as:

```
<plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-shade-plugin</artifactId>
    <version>2.3</version>
    <executions>
        <execution>
            <phase>package</phase>
            <goals>
                <goal>shade</goal>
            </goals>
            <configuration>
                <relocations>
                    <relocation>
                        <pattern>com.foo.bar</pattern>
                        <shadedPattern>somewhereelse.com.foo.bar</shadedPattern>
                    </relocation>
                </relocations>
            </configuration>
        </execution>
    </executions>
</plugin>
```

This interacts badly to cause the whole pom to be duplicated a number of times. This can be fixed by setting the scope to "/project/distributionManagement/relocation", however I don't know if this will have any unintended side effects.

It seems that there are a number of quite loose matchers there that could interact badly with any pom that contains third party plugin that uses "exclusion", "extension", "parent", "plugin" or "relocation" in their configuration.

**Comment by andham** [comment]
"extension" has now been handle to fix issue #7.
I agree with you that the matchers are too loose and should be more specific.

**Comment by andham** [comment]
This is superceded by issue #14.
